index.js

```diff
@@ -220,6 +220,15 @@ function onStream (opts, client, onerror) {
       next()
     })
 
+    if (opts.payloadLogFile) {
+      if (!client._payloadLogFile) {
+        client._payloadLogFile = require('fs').createWriteStream(opts.payloadLogFile, {flags: 'a'})
+      }
+      stream.on('data', function (chunk) {
+        client._payloadLogFile.write(chunk)
+      })
```

Comment:
If I used pump it would close the file when `stream` ended. That's not an issue as such, but then I couldn't re-use the file handle and would have to open it again for each stream that was opened

Reply:
But maybe that's cleaner? Even though there's an overhead of re-opening the file for each stream. As it's just for debug purposes, I'm not that concerned with the performance.

Reply:
Didn't think about the closing issue. In that case, we just leave it as-is.